File: m6-for/repetir_nome.js
import { question } from 'readline-sync'

function main(){
  const nome = question('Qual seu nome? ')
  const qtd = Number(question('Quantas vezes? '))
  
  // C-Like For, For de 3 Expressões, For Clássico, For Antigo
  for (let i = 0; i < qtd; i += 1){
    console.log(` ${i} > ${nome}`)
  }

}


main()

File: m6-for/repetir_nome.py
def main():
  nome = input('Qual seu nome? ')
  qtd = int(input('Quantas vezes? '))

  # for i in [1,2,3,4,5,6,7]:
  for i in range(0, qtd, 1):
    print(f'{i} > {nome}')


main()